config/vlan.py

```diff
@@ -74,57 +99,74 @@ def delete_db_entry(entry_name, db_connector, db_name):
 
 
 @vlan.command('del')
-@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('vid', metavar='<vid>', required=True)
+@click.option('-m', '--multiple', is_flag=True, help="Add Multiple Vlan(s) in Range or in Comma separated list")
```

Comment:
same as above

Reply:
Please note mutliple_vlan_parser will handle vlan(single, comma separated, range) on that list. This vid is exisiting behavior of VLANs. We don't modify existing behavior and added our proposed changes

Reply:
Why do we need the `--multiple` flag if the parser can already handle all possible inputs?

Reply:
Please note that --multiple flag is required as we did not add our new functionality to the existing command to keep intact the existing VLAN behavior/command.